Plots/XY/NCL_xy_5.py

```diff
@@ -0,0 +1,79 @@
+"""
+NCL_xy_5.py
+===============
+This script illustrates the following concepts:
+   - Adding a separate curve to an XY plot using gsn_polyline
+   - Drawing a Y reference line in an XY plot
+   - Filling the areas of an XY curve above and below a reference line
+   - Changing the width and height of a plot
+   - Creating a new date array to use in a plot
+   - Using named colors to indicate a fill color
+   - Changing the title on the Y axis
+   - Creating a main title
+   - Changing the size/shape of an XY plot using viewport resources
+   - Setting the mininum/maximum value of the Y axis in an XY plot
+
+See following URLs to see the reproduced NCL plot & script:
+    - Original NCL script: https://www.ncl.ucar.edu/Applications/Scripts/xy_5.ncl
+    - Original NCL plot: https://www.ncl.ucar.edu/Applications/Images/xy_5_1_lg.png
+"""
+
+###############################################################################
+# Import packages:
+import numpy as np
+import xarray as xr
+import matplotlib.pyplot as plt
+
+import geocat.datafiles as gdf
+from geocat.viz import util as gvutil
+
+###############################################################################
+# Read in data:
+
+# Open a netCDF data file using xarray default engine and load the data into xarrays
+ds = xr.open_dataset(gdf.get("netcdf_files/soi.nc"))
+dsoik = ds.DSOI_KET
+dsoid = ds.DSOI_DEC
+date = ds.date
+
+# Creating a new date array for x axis labels
+datedim = np.shape(date)[0]
+new_date = np.empty_like(date)
+
+for n in np.arange(0, datedim, 1):
+    yyyy = date[n]/100
+    mon = date[n]-yyyy*100
+    new_date[n] = yyyy + (mon-1)/12
+
+###############################################################################
+# Plot:
+
+# Generate figure (set its size (width, height) in inches) and axes
+plt.figure(figsize=(8, 4))
+ax = plt.gca()
+
+# Plot reference line
+plt.plot([0, datedim], [0, 0], color='grey', linewidth=0.75)
+
+# Plot data
```

Comment:
I could be wrong, but it looks like in the original NCL graph, the dsoik line is plotted in addition to the regions being filled. Maybe add this line and then change linewidth to match the thin black borders around the colored regions on the NCL graph:

```suggestion
# Plot data
dsoik.plot(ax=ax, color='black', linewidth=0.75)
```